tests/models/test_work.py

```diff
@@ -399,6 +399,25 @@ def test_calculate_presentation_sets_presentation_ready_based_on_content(self):
         work.calculate_presentation()
         eq_(True, work.presentation_ready)
 
+    def test_calculate_presentation_uses_default_audience_set_as_collection_setting(self):
+        default_audience = u"Audience"
```

Comment:
One more change: use a real audience here, like Classifier.AUDIENCE_ADULT. Eventually I'd like to make `Work.audience` an enumerated type and I'd rather not have tests where it's set to out-of-band strings.